api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -212,7 +212,7 @@ public async Task<IActionResult> UpdateAsync(ContentModel model, bool index = fa
 
             await _kafkaMessenger.SendMessageAsync(_kafkaHubOptions.HubTopic, new KafkaHubMessage(HubEvent.SendAll, new InvocationMessage("Content", new[] { new ContentMessageModel(content) })));
         }
-        else
+        else if (index)
```

Comment:
The Content Service doesn't always index content, this error shouldn't be displayed when indexing isn't requested.